### DISCUSSION TITLE
Storage link doesn’t work on Hostinger.com

### DISCUSSION THREAD

**Discussion by RenderWarrior** [PracticalBooks/Practical-Laravel#6] [Q&A] [answered]
I completed the book and I tried to deploy the site on Hostinger site.
I followed their tutorial for laravel8 but it’s not correct for the site developed, but I fixed it.
My only problem is about the products images… i cannot get the storage of laravel. The tutorial tells about putting a ‘laravel’ folder at the same level of the public_html folder and to move the public folder from laravel to the public_html.
But the site searches for the /storage folder on the public_html folder, but this doesn’t links to the laravel storage folder…

**Comment by RenderWarrior** [comment]
ok, I fixed it by myself. Everything works! :)

**Reply by danielgara** [reply]
@RenderWarrior great to hear that it worked for you!

**Reply by RenderWarrior** [reply, marked as answer]
Yes, i created a wrong symlink, then I understood how to create the right one.
